src/cljs/rems/administration/blacklist.cljs

```diff
@@ -64,10 +68,11 @@
 
 (rf/reg-event-fx
  ::fetch-users
- (fn []
-   (fetch "/api/blacklist/users"
-          {:handler #(rf/dispatch [::fetch-users-result %])
-           :error-handler (flash-message/default-error-handler :top "Fetch users")})
+ (fn [{:keys [db]} _]
+   (when (show-blacklist-add? (get-in db [:identity :roles]))
```

Comment:
This puts the ownership of the roles in this ns where it belongs, but there is an extra dispatch from the user side to here. The other places seem to so far avoid the dispatch entirely at the use site (i.e. never call `::fetch-users`). I'm not sure if one is better but maybe good to follow the same pattern everywhere? Ideally when you use a certain stateful component it could fetch its own data and check the rights too.

Reply:
rems.administration.blacklist is an experiment in modularizing components a bit more, so I just followed the existing architecture